src/utils.ts

```diff
@@ -1860,15 +1860,47 @@ export class newText extends newShape {
   }
 }
 
+export interface pointStyle {
+  size?: number, fillStyle?: string, strokeStyle?: string, marker?: string, markerOrientation?: string
+}
+
+export class newPointStyle implements pointStyle {
```

Comment:
When you'll change the newX to just X in the future, this will get weird, as pointStyle is already an interface.
I have no good convention to propose for interface/class implementation. Some people like IPointStyle. I would probably go for PointStyleInterface.

As a note, conventions state that interface names are normally PascalCase, so in any case, pointStyle should normally be PointStyle

Reply:
Changed to PointStyleInterface